click_completion/core.py

```diff
@@ -110,7 +110,10 @@ def get_choices(cli, prog_name, args, incomplete):
                     break
     choices = []
     if optctx:
-        choices += [c if isinstance(c, tuple) else (c, None) for c in optctx.type.complete(ctx, incomplete)]
+        choices += [c if isinstance(c, tuple) else(c, None) for c in optctx.type.complete(ctx, incomplete)]
```

Comment:
Please fix the missing space:
```suggestion
        choices += [c if isinstance(c, tuple) else (c, None) for c in optctx.type.complete(ctx, incomplete)]
```